x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/enrich/action/EnrichStatsAction.java

```diff
@@ -24,7 +24,8 @@
 public class EnrichStatsAction extends ActionType<EnrichStatsAction.Response> {
 
     public static final EnrichStatsAction INSTANCE = new EnrichStatsAction();
-    public static final String NAME = "cluster:admin/xpack/enrich/stats";
+    public static final String NAME = "cluster:monitor/xpack/enrich/stats";
+    public static final String BWC_NAME = "cluster:admin/xpack/enrich/stats";
```

Comment:
Can you add a javadoc here that it had this name in pre 7.8 versions

Reply:
Done